Source/Fuse.Platform/iOS/SystemUI.uno

```diff
@@ -211,6 +212,23 @@ namespace Fuse.Platform
 
 			var args = new SystemUIWillResizeEventArgs(SystemUIID.TopFrame, reason, endFrame, startFrame, animationDuration, 1);
 
+			Rect newFrameRect;
+			// when the endFrame size is > 40, we are streching to fill the space
+			if (endFrame.Size.Y > 40) 
```

Comment:
The magic number 40 again. Use a `const` with a name, and explanation, or why we have this value.